app/_components/HamburgerMenu.tsx

```diff
@@ -0,0 +1,191 @@
+"use client";
+import { useState } from "react";
+import Link from "next/link";
+import Image from "next/image";
+import { ExitCrossFC } from "../_atoms/Icons";
+
+const HamburgerMenu = () => {
+  const [menuOpen, setMenuOpen] = useState(false);
+
+  const handleNav = () => {
+    setMenuOpen(!menuOpen);
+  };
+  return (
+    <div className="flex items-center gap-4 relative z-10">
+      <button
+        onClick={handleNav}
+        className="block rounded p-2.5 text-accent transition hover:text-gray-600/75 md:hidden"
+      >
+        <span className="sr-only">Toggle menu</span>
+        <svg
```

Comment:
Could you move this SVG into the Icon.tsx component/atoms  and then call it in this file? This would help keep the code organised and reusable.